datacenterlight/views.py

```diff
@@ -229,6 +244,28 @@ def post(self, request):
         email = request.POST.get('email')
         name_field = forms.CharField()
         email_field = forms.EmailField()
+
+        try:
+            cores = cores_field.clean(cores)
+        except ValidationError as err:
+            msg = '{} {}.'.format(cores, _('is not a proper cores number'))
+            messages.add_message(self.request, messages.ERROR, msg, extra_tags='cores')
+            return HttpResponseRedirect(reverse('datacenterlight:index') + "#order_form")
+
+        try:
+            memory = memory_field.clean(memory)
+        except ValidationError as err:
+            msg = '{} {}.'.format(memory, _('is not a proper memory number'))
```

Comment:
I think we could use the text in the `err` object @siarheipuhach ? We are already raising it as 

`raise ValidationError(_('Not a proper ram number'))`

What do you think?

The same for other cases